### PR TITLE
translate philosophy.md to chinese

### PR DESCRIPTION
**Pull request by ji-huazhong**
# What does this PR do?

Part of https://github.com/huggingface/transformers/issues/26803

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

cc @stevhliu @jiaqiw09

### CODE REVIEW

**Review by jiaqiw09 — APPROVED** [review]

File: docs/source/zh/philosophy.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+该库围绕每个模型的三类类构建：
+
+- **模型类** 可以是 PyTorch 模型（[torch.nn.Module](https://pytorch.org/docs/stable/nn.html#torch.nn.Module)）、Keras 模型（[tf.keras.Model](https://www.tensorflow.org/api_docs/python/tf/keras/Model)）或 JAX/Flax 模型（[flax.linen.Module](https://flax.readthedocs.io/en/latest/api_reference/flax.linen/module.html)），这些模型可以使用库中提供的预训练权重。
+- **配置类** 存储构建模型所需的超参数（如层数和隐藏大小）。通常情况下，如果您使用不进行任何修改的预训练模型，则创建模型将自动处理配置的实例化（配置是模型的一部分）。
+- **预处理类** 将原始数据转换为模型可接受的格式。一个 [tokenizer](main_classes/tokenizer) 存储每个模型的词汇表，并提供编码和解码字符串为要馈送到模型的令牌嵌入索引列表的方法。[Image processors](main_classes/image_processor) 预处理视觉输入，[feature extractors](main_classes/feature_extractor) 预处理音频输入，而 [processor](main_classes/processors) 则处理多模态输入。
+
+所有这些类都可以从预训练实例中实例化、本地保存，并通过以下三种方法与 Hub 共享：
+
+- `from_pretrained()` 允许您从库自身提供的预训练版本或用户本地（或服务器上）存储的版本实例化模型、配置和预处理类。
```
Comment (stevhliu) [inline docs/source/zh/philosophy.md:65]: I believe there should be a link (https://huggingface.co/models) to the Hub here

**Review by stevhliu — APPROVED** [review]
Welcome back, good to see your translation PRs again! 🤗 

LGTM except for one missing link

**Comment by statelesshz** [comment]
the PR is ready to be merged. cc @stevhliu

File: docs/source/zh/philosophy.md (hunk 1 above)
Comment (stevhliu) [reply docs/source/zh/philosophy.md:65]: Let's add the missing link here, then merge

Comment (statelesshz) [reply docs/source/zh/philosophy.md]: Sorry for missing this comment, done in [c45f4c1](https://github.com/huggingface/transformers/pull/32177/commits/c45f4c159bfec2fd9fca102f5295f319b1849e3d).

**Pull request merged**